Editor/EditorCore/ProBuilderEditor.cs

```diff
@@ -567,7 +567,18 @@ void OnSceneGUI(SceneView sceneView)
 
         internal void HandleMouseEvent(SceneView sceneView, int controlID)
         {
-            if(m_CurrentEvent.type == EventType.MouseDown && HandleUtility.nearestControl == controlID)
+            if ((Event.current.modifiers & EventModifiers.Alt) == EventModifiers.Alt)
```

Comment:
There is some code in this function that handles exiting drag events. What happens if during a drag the `alt` key is held and a mouse button up event occurs? Does the drag exit successfully?

Reply:
I've added, in [a7935ea](https://github.com/Unity-Technologies/com.unity.probuilder/pull/487/commits/a7935ea69bab0ff81f2bef19eebaeab31d355a32), a check for `m_IsDragging` in that `if` statement to make sure that the rect selection is not interrupted if the `alt` key is pressed in the middle of the rect selection.